src/sidebar/components/MarkdownView.tsx

```diff
@@ -107,7 +107,10 @@ export default function MarkdownView(props: MarkdownViewProps) {
   }, [markdown]);
 
   useEffect(() => {
-    elementToMentionMap.current = renderMentionTags(content.current!, mentions);
+    elementToMentionMap.current = processAndReplaceMentionTags(
```

Comment:
To be pedantic, this is replacing mention _elements_ (as in `HTMLElement`) rather than tags (as in "tags in an HTML string")